catalyst/tools/settings.py

```diff
@@ -39,6 +39,7 @@ def __init__(
         use_libjpeg_turbo: bool = False,
         nmslib_required: Optional[bool] = None,
         transformers_required: Optional[bool] = None,
+        is_pytorch_above_v14: Optional[bool] = None,
```

Comment:
not sure if this is correct, in our case with torch version this should work like `IS_XLA_AVAILABLE`, isn't it?
you just cann't turn on/off torch version :)